apps/qubit/modules/settings/templates/diacriticsSuccess.php

```diff
@@ -0,0 +1,59 @@
+<?php decorate_with('layout_2col.php'); ?>
+
+<?php slot('sidebar'); ?>
+
+  <?php echo get_component('settings', 'menu'); ?>
+
+<?php end_slot(); ?>
+
+<?php slot('title'); ?>
+    <h1><?php echo __('Diacritics settings'); ?></h1>
+<?php end_slot(); ?>
+
+<?php slot('content'); ?>
+
+  <div class="alert alert-info">
+    <p><?php echo __('Please rebuild the search index if you are using diacritics mappings.'); ?></p>
```

Comment:
Good thinking to add this as a reminder given that folks may miss the flash message about it. Maybe specify that this needs to be done after the mapping is uploaded.